### PR TITLE
Add FAQ about rules to README

### PR DESCRIPTION
**Pull request by kraklin**
Adding FAQ to answer the question from issue #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,19 @@ those has its own rules for how it pluralizes. Check them
...
+## FAQ
+
+### Where does the rules come from?
+
+You're supposed to make your own dictionary of Rules and add words into it as you start using them in the app.
+
+Each language has its own ways of pluralizing words and we've opted out to let you "grow your own dictionary" based on the words you're using in your app, because:
+
+  - it would be impractical for the package to contain a whole dictionary in that given language,
+  - and I believe it would be impossible / very impractical to try to implement that behaviour with some kind of function. This is not a NLP package :)
```
Comment (Janiczek) [inline README.md]: I believe -> we believe

**Review by janjelinek — APPROVED** [review]

**Review by Janiczek — APPROVED** [review]
:+1:

File: README.md [hunk 2]
```diff
@@ -43,3 +43,19 @@ those has its own rules for how it pluralizes. Check them
 * Czech
 * French
 * Dutch
+
+## FAQ
+
+### Where does the rules come from?
```
Comment (Janiczek) [inline README.md]: Sorry one more thing!
```suggestion
### Where do the rules come from?
```

**Comment by kraklin** [comment]
I can't merge this PR, that's probably only available for you @Janiczek so feel free to do it any time

**Comment by Janiczek** [comment]
That's weird, I'll look at the repo settings @kraklin , thanks for notifying me

**Pull request merged**